node/services/service.go

```diff
@@ -0,0 +1,22 @@
+package service
```

Comment:
the package name must be consistent with the directory name.
currently, `service` != `services`
Keep everything as `services`

Reply:
@vgonkivs

Reply:
@Wondertan , fixed